.circleci/config.yml

```diff
@@ -378,15 +359,28 @@ jobs:
           test_apk_path: purchases/test_artifacts/integrationTest-test.apk
 
   run-firebase-tests-purchases-load-shedder-integration-test:
-    description: "Run purchases module integration tests for Android in Firebase using the load shedder servers. Variant integrationTest"
-    executor: gcp-cli/google
+    description: "Run purchases module integration tests for Android in Firebase using the load shedder servers"
+    <<: *android-executor
     steps:
       - checkout
-      - attach_workspace:
-          at: .
-      - run-firebase-tests:
-          app_apk_path: purchases/test_artifacts/loadShedderIntegrationTest-app.apk
-          test_apk_path: purchases/test_artifacts/loadShedderIntegrationTest-test.apk
+      - install-sdkman
+      - revenuecat/install-gem-unix-dependencies:
+          cache-version: v1
+      - gcp-cli/install
+      - gcp-cli/initialize:
+          gcloud-service-key: GCLOUD_SERVICE_KEY
+          google-compute-zone: GOOGLE_COMPUTE_ZONE
+          google-project-id: GOOGLE_PROJECT_ID
+      - android/restore-build-cache
+      - run:
+          name: Build and run load shedder integration tests
+          command: |
+            bundle exec fastlane android run_load_shedder_purchases_integration_tests
```

Comment:
Now all the logic (except installing the gcloud cli and initializing it) lives in fastlane, and it's much easier to send slack notifications.